hostplumber/pkg/hoststate/hoststate.go

```diff
@@ -134,10 +135,25 @@ func (hni *HostNetworkInfo) addNetPciDevice(devicePath, ifName string) error {
 	}
 	ifStatus.MTU = mtu
 
+	ipv4Addrs, err := iputils.GetIpv4Cidr(ifName)
+	if err != nil || len(*ipv4Addrs) == 0 {
+		hni.log.Infow("Error getting IPv4 for interface", "err", err, "ifName", ifName, "ipv4Addrs", *ipv4Addrs)
+	} else {
+		ifStatus.IPv4 = new(plumberv1.IPv4Info)
+		ifStatus.IPv4.Address = *ipv4Addrs
+	}
+
+	ipv6Addrs, err := iputils.GetIpv6Cidr(ifName)
+	if err != nil || len(*ipv6Addrs) == 0 {
+		hni.log.Infow("Error getting IPv6 for interface", "err", err, "ifName", ifName, "ipv6Addrs", *ipv6Addrs)
```

Comment:
Minor nit: it seems that not having a ipv6 (or ipv4) is an expected use case, maybe change the wording in these logs to not say "Error"—it might be confusing to people reading the logs otherwise.